move/example/sources/its/its.move

```diff
@@ -0,0 +1,211 @@
+module example::its_example {
```

Comment:
try adding the move formatter and run again (ping mysten team if there's an issue, maybe they have a newer version)